ts/store/reducers/wallet/transactions.ts

```diff
@@ -39,6 +43,20 @@ const getTransactions = (state: GlobalState) =>
       ) as ReadonlyArray<Transaction>
   );
 
+// Search transaction number in Set of read transactions
+export const isReadTransaction = (state: GlobalState, txid: number) =>
+  state.wallet.transactions.read.has(txid);
+
+// filter only unread transactions to account for the residual number
+export const getUnreadTransactions = (state: GlobalState) =>
+  pot.map(
+    state.wallet.transactions.transactions,
+    txs =>
+      values(txs).filter(
+        _ => _ !== undefined && !isReadTransaction(state, _.id)
+      ) as ReadonlyArray<Transaction> // ridurre a array di ID
```

Comment:
Why you need this cast?

Reply:
To get the right type in the mapStateToProps for the defined prop passed to TransactionList component. 
However, I moved it in the mapStateToProps and left cleaner the selector (in the last commit)